akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatorMessageSerializer.scala

```diff
@@ -268,7 +268,7 @@ class ReplicatorMessageSerializer(val system: ExtendedActorSystem)
     val status = dm.Status.parseFrom(bytes)
     Status(
       status.getEntriesList.asScala.map(e ⇒
-        e.getKey → AkkaByteString(e.getDigest.toByteArray()))(breakOut),
+        e.getKey → AkkaByteString.fromArrayUnsafe(e.getDigest.toByteArray()))(breakOut),
```

Comment:
getDigest returns a protobuf byte string, and toByteArray on that allocates a new array and copies its content into it.